borg/archiver.py

```diff
@@ -1353,6 +1398,11 @@ def build_parser(self, args=None, prog=None):
         subparser.add_argument('--keep-tag-files', dest='keep_tag_files',
                                action='store_true', default=False,
                                help='keep tag files of excluded caches/directories')
+        subparser.add_argument('--pattern', action=ArgparsePatternAction,
+                               metavar="PATTERN", help='include/exclude paths matching PATTERN')
```

Comment:
of course it would be great if we have same words for short and long options.

match and -M sounds good, but then the other options would be "--matches-from"?

"match" could be taken as action and mean positive and negative matching, but --matches-from sounds wrong because matches is a noun there and it does not contain matches, but patterns for matching.

so, guess we did not find best wording yet.